src/test/rgw/bucket_notification/bntests.conf.SAMPLE

```diff
@@ -2,6 +2,7 @@
 port = 8000
 host = localhost
 zonegroup = default
+zonename = default
```

Comment:
for the test to pass in teuthology, the following code should be modified:
https://github.com/ceph/ceph/blob/main/qa/tasks/notification_tests.py#L294

to include the new "zonename" parameter